### PR TITLE
Bug fix for not connected routes in SLD Editor

### PR DESCRIPTION
**Pull request by Flurb**

### CODE REVIEW

File: src/editors/SingleLineDiagram.ts [hunk 1]
```diff
@@ -15,14 +15,15 @@ import {
   getParentElementName,
   getAbsolutePositionWithCustomCoordinates,
   SVG_GRID_SIZE,
-  drawRoute,
+  drawRouteBetweenElements as drawRoute,
```
Comment (JakobVogelsang) [inline src/editors/SingleLineDiagram.ts:18]: Why not change both the names. Why keeping drawRoute as function name if you think the other one is better?

**Review by JakobVogelsang — COMMENTED** [review]
Can be merged. The question a raised as a matter of taste

File: src/editors/SingleLineDiagram.ts (hunk 1 above)
Comment (Flurb) [reply src/editors/SingleLineDiagram.ts:18]: Hi @JakobVogelsang , that's a weird change indeed.
I think it is something Visual Studio did, I will change it!

**Review by JakobVogelsang — APPROVED** [review]
LGTM thanks!

**Comment by JakobVogelsang** [comment]
I will rebase and merge first thing tomorrow @Flurb. Thanks again for this nice editor

**Pull request merged**